packages/main/src/TimePickerBase.ts

```diff
@@ -243,11 +301,81 @@ class TimePickerBase extends UI5Element {
 		this._isPickerOpen = false;
 	}
 
-	async _handleInputClick() {
+	async onResponsivePopoverAfterOpen() {
+		this._isPickerOpen = true;
+		const responsivePopover = await this._getPopover();
+		responsivePopover.querySelector<TimeSelectionClocks>("[ui5-time-selection-clocks]")!._focusFirstButton();
+	}
+
+	/**
+	 * Opens the Inputs popover.
+	 * @async
+	 * @public
+	 * @method
+	 * @name sap.ui.webc.main.TimePickerBase#openInputsPopover
+	 * @returns {Promise} Resolves when the picker is open
+	 */
+	async openInputsPopover() {
```

Comment:
Both popovers are sharing the same logic for open / close / submit with very little changes. Can we combine somehow the shared logic like passing the corresponding  popover or boolean function parameter?

Reply:
It is not so easy, and as result it will move the same logic but with if/else statements... In addition - these methods are public, and adding parameter (even if it is boolean or popover), will lead to API change. So for now I think it is fine as it is.